src/index.html

```diff
@@ -1,13 +1,363 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Hello Mate Academy</h1>
+  <body class="page__body">
+    <aside class="page__menu">
+      <div class="menu" id="menu">
+        <a
+          href="#"
+          class="menu__close page__icon"
+        ></a>
+
+        <nav class="menu__nav">
+          <ul class="menu__list">
+            <div class="menu__wrap">
+              <li class="menu__item">
+                <a class="menu__link" href="#gallery">
+                  Галерея
+                </a>
+              </li>
+              <li class="menu__item">
+                <a class="menu__link" href="#events">
+                  Виставки та події
+                </a>
+              </li>
+            </div>
+            <div class="menu__wrap">
+              <li class="menu__item">
+                <a class="menu__link" href="#subscription">
+                  Про Нас
+                </a>
+              </li>
+              <li class="menu__item">
+                <a class="menu__link" href="#contacts">
+                  Контакти
+                </a>
+              </li>
+            </div>
+
+          </ul>
+        </nav>
+
+        <div class="menu__grafic grafic">
+          <p class="grafic__title">
+            Графік роботи
+          </p>
+
+          <div class="grafic__week">
+            <p><span class="grafic__bold">12:00 — 20:00</span> середа</p>
+            <p><span class="grafic__bold">12:00 — 20:00</span> четвер</p>
+            <p><span class="grafic__bold">12:00 — 20:00</span> п’ятниця</p>
+            <p><span class="grafic__bold">11:00 — 19:00</span> субота</p>
+            <p><span class="grafic__bold">11:00 — 19:00</span> неділя</p>
+          </div>
+
+
+          <div class="grafic__kasa">
+            <p><span class="grafic__bold">Вихідні:</span> понеділок, вівторок
+            <br>Каса припиняє роботу за 1 годину до закриття музею</p>
+          </div>
+        </div>
+      </div>
+    </aside>
+
+    <header class="header page__header page__container" id="header">
+      <a
+        href="#menu"
+        class="header__menu page__icon"
+      ></a>
+
+      <a href="#" class="header__logo">
+        <img
+          src="./images/icons/namu-logo.svg"
+          alt="namu logo"
+          class="header__img"
+        >
+      </a>
+    </header>
+
+    <a href="#" class="page__wrapper"></a>
+
+    <main class="page__main">
+      <section id="museum" class="museum">
+        <div class="museum__top">
+        </div>
+
+        <div class="museum__bottom">
+          <h1 class="museum__title">ХУДОЖНІЙ МУЗЕЙ</h1>
+          <a href="#" class="tickets museum__tickets">
+            <div class="tickets__text">
+              Квитки
+            </div>
+            <div class="tickets__arrow">
+              <img
+              src="./images/icons/arrow.svg"
+              alt="arrow"
+              class="tickets__image"
+            >
```

Comment:
Need to fix and check it everywhere
```suggestion
              <img
                src="./images/icons/arrow.svg"
                alt="arrow"
                class="tickets__image"
              >
```